src/multipatch/connectivity/matching_idx_slice.hpp

```diff
@@ -224,4 +224,4 @@ class MatchingIdxSlice
         IdxStep<Grid1D> size((idx_range.size() - 1) / stride.value() + 1);
         return IdxRangeSlice<Grid1D>(idx_front, size, stride);
     }
-};
\ No newline at end of file
+};
```

Comment:
I would suggest to update to clang-format 16 at some point. It will handle it https://releases.llvm.org/16.0.0/tools/clang/docs/ClangFormatStyleOptions.html#insertnewlineateof

Reply:
The missing newline at the end of file.

Reply:
The previously missing newline 

Maybe. I think changing clang-format leads to an enormous amount of changes in the code